install/debian/postinst.in

```diff
@@ -20,21 +20,27 @@ case "$1" in
     # Create directories
     install -d -m 755 -o $NAME -g $NAME $CLIENT_HOME $CLIENT_LOGS
 
-    # Create a symbolic link to the configuration file
-    ln -sf /etc/fahclient/config.xml $CLIENT_HOME/config.xml
-
-    # Remove old unit if it exists
-    rm -f /etc/systemd/system/${NAME}.service
+    # migrate v7 configuration
+    # only copy if destination does not exist or is a symlink,
+    # as installed by previous v8 packages
+    if [ -f /etc/fahclient/config.xml ] && \
+      ([ -L $CLIENT_HOME/config.xml ] || [ ! -e $CLIENT_HOME/config.xml ]); then
+      cp --remove-destination /etc/fahclient/config.xml $CLIENT_HOME/
+      chown $NAME:$NAME $CLIENT_HOME/config.xml
+      chmod 0644 $CLIENT_HOME/config.xml
+      echo
+      echo "v7 client configuration copied to $CLIENT_HOME/config.xml"
+      echo
+      echo "It's recommended to purge the old package to remove leftover files:"
+      echo
+      echo "  sudo apt purge fahclient"
+      echo
```

Comment:
Please drop this blank `echo`, now that `systemctl enable` below is silent.

Reply:
Done.